eng/azure-sdk-deps.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <PublishDepsFilePath>$(IntermediateOutputPath)\azure-sdk-deps.json</PublishDepsFilePath>
+    <SdkRootPath>..\sdk</SdkRootPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ExcludeProjects Include="$(SdkRootPath)\**\*Test*.csproj" />
```

Comment:
eng/service.proj uses pattern `sdk\*\**\tests\**\*.csproj`. Shouldn't we match to be safe and select all the same stuff? In fact, why not just import/use service.proj for consistency and dedup'ing?

Reply:
The patterns are definitely slightly different and I agree I don't like that but I made this pattern much more targeted to only look at things in the src folder to avoid most of the other stuff.

I tried to import service.proj but msbuild isn't happy because they are conflicting project types and duplicate imports that occur. If we end up keeping this maybe I can factor out the project selection into a copy props file and share it that way, but looking at the other analyze dependency steps I think I will switch this to instead reference the packages we are analyzing instead of using project references.